lib/topological_inventory/ansible_tower/operations/source.rb

```diff
@@ -59,13 +72,14 @@ def connection_status
         end
 
         def connection_check
+          check_time
```

Comment:
Is this here to preload `@check_time`?

Reply:
yes, to ensure the time is grabbed before checking and to be the same for both timestamps